war/src/main/java/io/lumeer/engine/controller/ConfigurationFacade.java

```diff
@@ -276,50 +295,86 @@ public void setUserConfigurationInteger(final String key, final int value) {
+    *
+    * @param key
+    *       the name of key
+    * @param value
+    *       the Integer value of the given key
+    */
+   public void setProjectConfigurationInteger(final String key, final int value) {
+      setConfiguration(ConfigurationLevel.PROJECT, key, value);
+   }
+
+   /**
+    * Sets a new key-DataDocument value to configuration entry for active project. If the given key exists, the document will be updated.
+    *
+    * @param key
+    *       the name of key
+    * @param document
+    *       the DataDocument of the given key
+    */
+   public void setProjectConfigurationDocument(final String key, final DataDocument document) {
+      setConfiguration(ConfigurationLevel.PROJECT, key, document);
    }
 
    /**
-    * Sets a new key-String value to configuration entry for currently logged team. If the given key exists, its value will be updated.
+    * Sets a new key-String value to configuration entry for currently logged organisation. If the given key exists, its value will be updated.
     *
     * @param key
     *       the name of key
     * @param value
     *       the String value of the given key
     */
-   public void setTeamConfigurationString(final String key, final String value) {
-      setConfiguration(false, key, value);
+   public void setOrganisationConfigurationString(final String key, final String value) {
+      setConfiguration(ConfigurationLevel.ORGANISATION, key, value);
    }
 
    /**
-    * Sets a new key-Integer value to configuration entry for currently logged team. If the given key exists, its value will be updated.
+    * Sets a new key-Integer value to configuration entry for currently logged organisation. If the given key exists, its value will be updated.
     *
     * @param key
     *       the name of key
     * @param value
     *       the Integer value of the given key
     */
-   public void setTeamConfigurationInteger(final String key, final int value) {
-      setConfiguration(false, key, value);
+   public void setOrganisationConfigurationInteger(final String key, final int value) {
+      setConfiguration(ConfigurationLevel.ORGANISATION, key, value);
    }
 
    /**
-    * Sets a new key-DataDocument value to configuration entry for currently logged team. If the given key exists, the document will be updated.
+    * Sets a new key-DataDocument value to configuration entry for currently logged organisation. If the given key exists, the document will be updated.
     *
     * @param key
     *       the name of key
     * @param document
     *       the DataDocument of the given key
     */
-   public void setTeamConfigurationDocument(final String key, final DataDocument document) {
-      setConfiguration(false, key, document);
+   public void setOrganisationConfigurationDocument(final String key, final DataDocument document) {
+      setConfiguration(ConfigurationLevel.ORGANISATION, key, document);
```

Comment:
I can understand why we have different getters for `String` and `Integer` values. But why do we have setters for different types when we can use just `Object`?